app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import shutil
+
+source = "/Users/jacktyler/py-copy-file/app/file.txt"
+destination = "/Users/jacktyler/py-copy-file/app/new_file.txt"
```

Comment:
You shouldn't put your local addresses into the code. It should be working with any address.